templates/parachain/node/src/cli.rs

```diff
@@ -66,6 +66,12 @@ pub struct Cli {
 	#[command(flatten)]
 	pub run: cumulus_client_cli::RunCmd,
 
+	/// EXPERIMENTAL: Use slot-based collator which can handle elastic scaling.
+	///
+	/// Use with care, this flag is unstable and subject to change.
+	#[arg(long)]
+	pub experimental_use_slot_based: bool,
```

Comment:
The slot based collator is just an implementation detail, which is likely not going to be of interest to node operators.
Why not just `--elastic-scaling` ? In the future slot based collator will be used for everything, but for the time being this is only the only way to enable elastic scaling.

Reply:
I mean this is in the end only for people who want to test the bleeding edge. The CLI flag will vanish at some point. 

If we go with elastic scaling we should at least keep `--experimental-elastic-scaling`.